Stationeers.VS.props

```diff
@@ -0,0 +1,50 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$(SolutionDir)Stationeers.VS.User.props" />
+
+  <PropertyGroup>
+    <TargetFramework>net4.8</TargetFramework>
+    <TargetFrameworkVersion>v4.8</TargetFrameworkVersion>
+    <LangVersion>9.0</LangVersion>
+    <ErrorReport>none</ErrorReport>
+    <LanguageTargets>$(MSBuildToolsPath)\Microsoft.CSharp.targets</LanguageTargets>
+
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
+    <AppendTargetFrameworkToOutputPath>false</AppendTargetFrameworkToOutputPath>
+    <PlatformTarget>AnyCPU</PlatformTarget>
+    <Targets>AnyCPU</Targets>
+    <OutputPath>$(SolutionDir)bin\$(Configuration)</OutputPath>
+
+    <!-- Do Not Change Anything Below -->
+    <StationeersDirectory>$(SteamLibraryDirectory)\Stationeers</StationeersDirectory>
```

Comment:
Is there a reason this isn't just a user prop?  I don't see any reference to SteamLibraryDirectory elseware and hardcoding this makes it harder for extracted setups.  (i.e. were the Modding stationers directory is extracted from the main folder so that the main Stationeers install can remain vanilla.)